docs/contributing.rst

```diff
@@ -0,0 +1,334 @@
+   menu to "my_github_handle/sunraster" and the "compare" drop-down to
+   "my_fix". This sets the repo and branch in which you have made the
+   changes you want to merge.
+#. Enter a title and a description of the PR in the appropriate
+   boxes.  Try to be descriptive so other developers can understand
+   the purpose of the PR.
+#. Finally, click the green "Create Pull Request" button.  Well done!
+   You've opened your first PR!
+
+Now begins the process of code review.  Code review is a standard
+industry practice which involves other members of the community
+reviewing your proposed changes and suggesting improvements.  It is a
+fantastic way of improving your coding abilities as well as preserving
+the integrity of the overall package.  A bugfix does not have
+to be finished in order to open a PR. In fact, most PRs are incomplete
+when they are first opened. This allows others to follow your progress
+and contribute suggestions if you get stuck.  Anyone can review a  PR.
+Experience is not a disqualifying factor.  But it is recommended that
+at least one experienced developer reviews your code. You can make
+updates to your PR by editing your local ``my_fix`` branch, committing
+the new changes and pushing them to the ``my_repo`` remote.  The PR
+will then be automatically updated with the new commits.  Once you've
+made all changes and the online tests have passed, those reviewing
+your code can approve the PR.  Approved PRs can then be merged by
+those with write permissions to the repo.  Congratulations!  You have
+just contributed to ``sunraster``!
+
+Be sure to pull your the newly contributed changes to your local
+master branch by doing:
+
+.. code-block:: console
+
+  git checkout master
+  git pull origin master
+
+You are now ready to start using the newly improved ``sunraster`` development
+version of ``sunraster``, including your changes!
+
+If you have questions about this guide or making contributions,
+``sunraster`` and SunPy developers are always happy to help.  See
+:ref:`getting_help`.  Happy coding and talk to you soon!
+
+.. _issue tracker: https://github.com/sunpy/sunraster/issues
+.. _sunraster GitHub repository: https://github.com/sunpy/sunraster
+.. _GitHub: https://github.com/
+.. _git: https://git-scm.com/
+.. _SunPy Newcomer's Guide: http://docs.sunpy.org/en/stable/dev_guide/newcomers.html
+.. _github guide: https://guides.github.com/
+.. _git guide: https://git-scm.com/book/en/v2/Getting-Started-Git-Basics
+.. _SunPy version control guide: http://docs.sunpy.org/en/stable/dev_guide/version_control.html
+.. _SunPy Developer's Guide: http://docs.sunpy.org/en/stable/dev_guide
+.. _pull requests: https://help.github.com/articles/about-pull-requests/
+.. _Git branches: https://git-scm.com/book/en/v2/Git-Branching-Branches-in-a-Nutshell
+.. _git branch: https://git-scm.com/book/en/v2/Git-Branching-Branches-in-a-Nutshell
+.. _git remote: https://git-scm.com/book/en/v2/Git-Basics-Working-with-Remotes
```

Comment:
Is it worth adding this?

Reply:
The link to git remote docs?  Any reason not to?

Reply:
Sorry, I meant this entire document.

Reply:
I don’t see a reason not to and it tells people how to report bugs etc. Why do you think it’s better removed?

Reply:
How/where to report bugs is fine, I mean more the git commands and the like.
The reason we lack them in the sunpy one is due to fact that other places (astropy and github) have some nicer guides that are kept more up to date than what we were doing. I am just worried about the age of this specific document.